httpie/cli/argtemplate.py

```diff
@@ -54,6 +54,10 @@ def store_json_template(args):
 
 
 def edit_json_template(args):
+    """
+    Edit a template in templates.json, the format for running this is:
+        http template <name> <item> <value>
```

Comment:
oh I missed this before but this should be `http editt ...`

Reply:
nice catch, fixed it!